pkg/controller/migtoken/validation.go

```diff
@@ -0,0 +1,47 @@
+package migtoken
+
+import (
+	migapi "github.com/konveyor/mig-controller/pkg/apis/migration/v1alpha1"
+)
+
```

Comment:
NIT: Consider using comments here to distinguish why these const's are in different blocks

Reply:
+1 The other controllers label the const blocks as:
// Types
// Reasons
// Messages
etc..